Quadriga/src/main/java/edu/asu/spring/quadriga/web/uploadtransformation/UploadTransformationController.java

```diff
@@ -0,0 +1,47 @@
+package edu.asu.spring.quadriga.web.uploadtransformation;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.spring.quadriga.service.uploadtransformation.ITransformationManager;
+
+/**
+ * 
+ * @author JayaVenkat
+ * This is a controller which takes request from uploadTransfomation.jsp to upload transformation files
+ */
+@Controller
+public class UploadTransformationController {
+
+	@Autowired
+	private ITransformationManager uploadTnfmManager;
+	
+	@RequestMapping(value="auth/transformation/upload",method=RequestMethod.POST)
+	public String uploadTransformationFiles(@ModelAttribute("UploadTransformationBackingBean") UploadTransformationBackingBean formBean, ModelMap map, 
```

Comment:
needs an access policy. only project owner, project admins, and project collaborators should be able to do this.